web/src/repositories/auth/dto/auth-login.dto.ts

```diff
@@ -0,0 +1,9 @@
+export interface AuthLoginRequestDTO {
+  email: string;
+  password: string;
+}
+
+export interface AuthLoginResponseDTO {
+  accessToken: string;
+  refreshToken: string;
+}
```

Comment:
Personnaly, I prefixed with `In` or `Out` my DTOs for the mobile app. Maybe do the same ? 
```suggestion
export interface OutAuthLoginDTO {
  email: string;
  password: string;
}

export interface InAuthLoginDTO {
  accessToken: string;
  refreshToken: string;
}
```